Dockerfile

```diff
@@ -131,6 +131,39 @@ RUN mkdir src && \
     cd ../ && rm -rf src &&\
     echo "${__prefix}/pythia6/" > /etc/ld.so.conf.d/pythia6.conf
 
+###############################################################################
+# LHAPDF
+#
+# Needed for GENIE
+#
+# - We disable the python subpackage because it is based on Python2 whose
+#   executable has been removed from Ubuntu 22.04.
+###############################################################################
+LABEL lhapdf.version="6.5.4"
+RUN mkdir src &&\
+    ${__wget} https://lhapdf.hepforge.org/downloads/?f=LHAPDF-6.5.4.tar.gz |\
+      ${__untar} &&\
+    cd src &&\
+    ./configure --disable-python --prefix=${__prefix} &&\
+    make -j$NPROC install &&\
+    cd ../ &&\
+    rm -rf src
+
+###############################################################################
+# PYTHIA8
+###############################################################################
```

Comment:
Why does Pythia8 require `rsync`? Does it download stuff from servers during running?